### DISCUSSION TITLE
import paddle error

### DISCUSSION THREAD

**Discussion by TUCJVXCB** [PaddlePaddle/PaddleOCR#12428] [Q&A] [answered]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：MacOS 10.15.7 Intel X86
- 版本号/Version：Paddle： 2.5.2  PaddleOCR：2.6.0.1 问题相关组件/Related components：
- 运行指令/Command Code：import paddle
- 完整报错/Complete Error Message：
```
ImportError: dlopen(/Users/jesus/anaconda3/envs/py39/lib/python3.9/site-packages/paddle/fluid/libpaddle.so, 2): Symbol not found: __ZNSt3__113basic_filebufIcNS_11char_traitsIcEEE5closeEv
  Referenced from: /Users/jesus/anaconda3/envs/py39/lib/python3.9/site-packages/paddle/fluid/libpaddle.so (which was built for Mac OS X 13.0)
  Expected in: /usr/lib/libc++.1.dylib
```

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): yes

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by CMWFireworm** [comment, marked as answer]
@TUCJVXCB I solved it by the command:
python -m pip install --force-reinstall paddlepaddle==2.5

https://stackoverflow.com/questions/76388016/error-can-not-import-paddle-core-while-this-file-exists/77544088#77544088
